src/torrent.cpp

```diff
@@ -1001,7 +1000,7 @@ namespace libtorrent
 		{
 			throw;
 		}
-		catch (system_error const& err)
+		catch (boost::system::system_error const& err)
```

Comment:
I thought there was a typedef pulling ``system_error`` into the libtorrent namespace, no?